benchmarks/benchmark.py

```diff
@@ -88,6 +88,7 @@
     'aggregate_report_rate': 'Throughput',
     'average': 'Average',
     'aggregate_report_median': 'Median',
+    'aggregate_report_90%_line': 'aggregate_report_90_line',
```

Comment:
Is this the only change required?

Reply:
Yep. This is a map of keys to be read from csv, and the corresponding header to be printed on the console. 

Eg : Run this ```python3 benchmarks/benchmark.py -l 10 -s --model noop-v1.0```

This gives the following output on my local machine : 

```
Running benchmark latency with model noop-v1.0
Output available at /tmp/MMSBenchmark/out/latency/noop-v1
Report generated at /tmp/MMSBenchmark/out/latency/noop-v1/report/index.html
[{'latency_noop-v1.0_Inference_Request_Average': 43,
  'latency_noop-v1.0_Inference_Request_Median': 5,
  'latency_noop-v1.0_Inference_Request_Throughput': 9.9,
  'latency_noop-v1.0_Inference_Request_aggregate_report_90_line': 8,
  'latency_noop-v1.0_Inference_Request_aggregate_report_99_line': 387,
  'latency_noop-v1.0_Inference_Request_aggregate_report_error': '0.00%'}]
```